script.js

```diff
@@ -0,0 +1,127 @@
+    altText: 'featured speaker image',
+    nameOfSpeaker: 'Nick',
+    designation: 'Founder of building homes',
+    history:
+      'Vast years of experience in Real Estate, nationally and internationally',
+  },
+  {
+    id: 5,
+    pic: 'https://images.pexels.com/photos/864994/pexels-photo-864994.jpeg?auto=compress&cs=tinysrgb&w=600.png',
+    altText: 'featured speaker image',
+    nameOfSpeaker: 'Racheal Mbabazi',
+    designation: 'CEO of lets Build.',
+    history:
+      'Six years of experience in interior design, nationally and internationally',
+  },
+  {
+    id: 6,
+    pic: 'https://images.pexels.com/photos/7642009/pexels-photo-7642009.jpeg?auto=compress&cs=tinysrgb&w=600.png',
+    altText: 'featured speaker image',
+    nameOfSpeaker: 'Phillip Musiime',
+    designation: 'Founder of Tomorrow World',
+    history:
+      'Vast years of experience in real estate, nationally and internationally.',
+  },
+];
+
+const speakerContainer = document.createElement('section');
+speakerContainer.setAttribute('class', 'featured-speakers');
+speakerContainer.setAttribute('id', 'speakers');
+data.forEach((speakerPara) => {
+  const speakerSection = document.createElement('div');
+  speakerSection.setAttribute('class', 'speaker-section');
+  speakerSection.setAttribute('id', 'each-speaker');
+
+  const imageSection = document.createElement('div');
+  imageSection.setAttribute('class', 'speaker-image-section');
+  const image = document.createElement('img');
+  image.setAttribute('class', 'speaker-image');
+  image.setAttribute('src', speakerPara.pic);
+  image.setAttribute('alt', speakerPara.altText);
+
+  imageSection.appendChild(image);
+  speakerSection.appendChild(imageSection);
+
+  const contentSection = document.createElement('div');
+  contentSection.setAttribute('class', 'speaker-details');
+  contentSection.setAttribute('id', 'about-speaker');
+
+  const speakerName = document.createElement('h3');
+  speakerName.setAttribute('class', 'speaker-name');
+  speakerName.textContent = speakerPara.nameOfSpeaker;
+  contentSection.appendChild(speakerName);
+
+  const speakerDesignation = document.createElement('p');
+  speakerDesignation.setAttribute('class', 'speaker-designation');
+  speakerDesignation.textContent = speakerPara.designation;
+  contentSection.appendChild(speakerDesignation);
+
+  const dotText = document.createElement('p');
+  dotText.setAttribute('id', 'dots');
+  dotText.textContent = '.........';
+  contentSection.appendChild(dotText);
+
+  const speakerHistory = document.createElement('p');
+  speakerHistory.setAttribute('class', 'speaker-history');
+  speakerHistory.textContent = speakerPara.history;
+  contentSection.appendChild(speakerHistory);
+
+  speakerSection.appendChild(contentSection);
+  speakerContainer.appendChild(speakerSection);
+});
+
+const body = document.querySelector('body');
+const partners = document.getElementById('partners');
+body.insertBefore(speakerContainer, partners);
```

Comment:
- [ ] Good Job for Implementing  the section "Featured speakers"  dynamically in JavaScript. But still the pages should look almost identical to the original design. So Kindly Work on styling page just Like the Design. Adjust page background color, Underline layout position and speakers background image just like the design.

The Template design Look likes this : 
 
![C1](https://user-images.githubusercontent.com/93598189/191452418-6d527996-cc58-47f4-af06-b690b99306e2.PNG)

Your Project Design for page Look Likes this : 
![C2](https://user-images.githubusercontent.com/93598189/191452544-8ca54690-3adc-4721-b1c0-c07ba7402e73.PNG)